aclk/agent_cloud_link.c

```diff
@@ -723,13 +723,58 @@ void aclk_del_collector(const char *hostname, const char *plugin_name, const cha
     }
 
     _free_collector(tmp_collector);
+
+}
+/*
+ * Take a buffer, encode it and rewrite it
```

Comment:
Looks like it doesn't rewrite it anymore?

Reply:
It rewrites it into a temporary buffer.